docs/source/ko/_toctree.yml

```diff
@@ -2,6 +2,11 @@
   sections:
   - local: index
     title: 소개
+- title: Conceptual guides
+  sections:
+  - local: conceptual_guides/react
+    title: 🤔 멀티스텝 에이전트는 어떻게 동작하나요?
+- title: Examples
```

Comment:
This is wrongly located: "installation" sub-section below should be located inside "Get started" section.